docs/client/operations.md

```diff
@@ -0,0 +1,177 @@
+# <img align="center" src="../images/logo.png">  Calling Operations with Your C# Client
+
+AutoRest provides both synchronous and asynchronous method overloads for each service operation.
+Depending on your swagger definition, operations can be accessed through operation groups (TODO: link to swagger docs) on the client,
```

Comment:
Does this TODO need an issue for tracking or was it expected to be handled in this PR?